src/Apps/Order/Components/AddressForm.tsx

```diff
@@ -84,6 +87,7 @@ export class AddressForm extends React.Component<
             <CountrySelect
               selected={this.state.address.country}
               onSelect={this.changeValueHandler("country")}
+              disabled={this.props.disabled}
```

Comment:
The `< CountrySelect >` component doesn't disable itself even when `disabled` is true, but this is because the `<LargeSelect>`component that's in Palette. I'll address this on the palette side and when we upgrade palette we'll get the disabled state for free.